commands/service_dev.go

```diff
@@ -79,7 +79,7 @@ func (c *serviceDevCmd) runE(cmd *cobra.Command, args []string) error {
 	defer func() {
 		var err error
 		pretty.Progress("Removing the service...", func() {
-			err = c.e.ServiceDelete(id)
+			err = c.e.ServiceDelete(false, id)
```

Comment:
What about adding the flag `--keep-data` to command dev to prevent the deletion of volume, and the default behavior will be to delete the volume?
Anyway, let's NOT do this change in this PR.